### PR TITLE
Create individual pipeline from pipelinerun to make it reusable

### PR DESCRIPTION
**Pull request by subodh101**
What?
Reference: https://github.com/kubeflow/testing/issues/769
- [x] Separate py test pipeline from run https://github.com/kubeflow/testing/issues/769#issuecomment-722390032.
- [x] Add python path as per the https://github.com/kubeflow/testing/issues/769#issuecomment-722407594.
- ~Add lint in the name as per the https://github.com/kubeflow/testing/pull/775#issuecomment-721947828.~

### CODE REVIEW

**Comment by subodh101** [comment]
mostly it is same.
I have just added lint in the name.
Tested it and confirms it works.

**Comment by PatrickXYS** [comment]
@subodh101 Thanks! I'm pretty sure we need pylint our code since we enabled py-unittest, so my question is what step does user need to do before sending PR and received error before not pylint their code?

Can we document somewhere or change the PR template say something like

```
make sure you finish these steps before sending PR

[All the steps that are required]
```

**Comment by subodh101** [comment]
> @subodh101 Thanks! I'm pretty sure we need pylint our code since we enabled py-unittest, so my question is what step does user need to do before sending PR and received error if not pylint their code?
> 
> Can we document somewhere or change the PR template say something like
> 
> ```
> make sure you finish these steps before sending PR
> 
> - pytest py/kubeflow/testing/py_lint_test.py -s --rcfile=.pylintrc
> ```

Oh. thank you. Yes, that sounds good. Let's do it in a separate PR.
I think we need to create an idea issues PR for what can be done to do faster development.

File: tekton/templates/pipelines/py-unittests-pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    params:
+    - name: artifacts-gcs
+      value: "$(params.artifacts-gcs)"
+    resources:
+      inputs:
+      - name: testing-repo
+        resource: testing-repo
+    taskRef:
+      name: py-test
+    runAfter:
```
Comment (jlewi) [inline tekton/templates/pipelines/py-unittests-pipeline.yaml:35]: Delete this. The unittests and lint can run in parallel which should be faster.

**Comment by jlewi** [comment]
Thanks for the quick fix.

**Comment by subodh101** [comment]
@jlewi It is running. Looks good now! Please merge.

**Pull request merged**